README.md

```diff
@@ -6,7 +6,7 @@ A website that showcases projects! **If you want to use it and it's not exactly
 fitting your purpose, please do make upstream changes. We expect this project
 to be of use to others and we're happy to make it more generic so it's easier
 to adapt to other organizations.** This was built by coala community, to view
-live projects visit [projects.coala.io](https://projects.coala.io)
+live projects visit [projects.coala.io](http://projects.coala.io)
```

Comment:
this should be https .  I think we've fixed that problem.  https://github.com/coala/projects/issues/604#issuecomment-408143355